packages/features/ee/workflows/lib/reminders/templates/customTemplate.ts

```diff
@@ -17,7 +17,14 @@ export type VariablesType = {
 };
 
 const customTemplate = async (text: string, variables: VariablesType, locale: string) => {
-  const timeWithTimeZone = `${variables.eventTime?.locale(locale).format("HH:mm")} (${variables.timeZone})`;
+  const translatedDate = new Intl.DateTimeFormat(locale, {
```

Comment:
using Intl now for date translations: 

English: 
![Screenshot 2023-02-08 at 10 25 32](https://user-images.githubusercontent.com/30310907/217575112-b70aea9d-998b-4949-9d4e-6cb102a47187.png)

German:
![Screenshot 2023-02-08 at 10 25 35](https://user-images.githubusercontent.com/30310907/217575133-dac46c2a-11e8-418e-9a0e-e97fdcd05d8e.png)